server/app/routes.py

```diff
@@ -11,423 +11,130 @@ def home():
-def create_invitation():
+@app.route('/supply_requests/<int:request_id>', methods=['PUT'])
+def update_supply_request(request_id):
     data = request.get_json()
-    email = data.get('email')
-    user_id = data.get('user_id')
-
-    if not email or not user_id:
-        return jsonify({
-            "status": "Failed",
-            "message": "Email and user_id are required.",
-            "data": None
-        }), 400
-
-    token = str(uuid.uuid4())
-    expiry_date = datetime.now(timezone.utc) + timedelta(days=7)
-
-    new_invitation = Invitation(
-        token=token,
-        email=email,
-        expiry_date=expiry_date,
-        user_id=user_id
-    )
-
-    try:
-        db.session.add(new_invitation)
-        db.session.commit()
-    except Exception as e:
-        db.session.rollback()
-        return jsonify({
-            "status": "Failed",
-            "message": "An error occurred while adding the invitation.",
-            "data": str(e)
-        }), 500
-
-    return jsonify({
-        "status": "Success",
-        "message": "Invitation created successfully.",
-        "data": new_invitation.to_dict()
-    }), 201
-
-
-@app.route('/invitations/<int:invitation_id>', methods=['PUT'])
-def update_invitation(invitation_id):
-    invitation = Invitation.query.get(invitation_id)
-    if not invitation:
-        return jsonify({
-            "status": "Failed",
-            "message": "Invitation not found.",
-            "data": None
-        }), 404
-
-    data = request.get_json()
-    invitation.token = data.get('token', invitation.token)
-    invitation.email = data.get('email', invitation.email)
-    invitation.expiry_date = data.get('expiry_date', invitation.expiry_date)
-    invitation.is_used = data.get('is_used', invitation.is_used)
-
+    req = SupplyRequest.query.get_or_404(request_id)
+    req.inventory_id = data['inventory_id']
+    req.user_id = data['user_id']
+    req.request_date = data['request_date']
+    req.status = data['status']
     db.session.commit()
-    return jsonify({
-        "status": "Success",
-        "message": "Invitation updated successfully.",
-        "data": invitation.to_dict()
-    }), 200
+    return jsonify({'message': 'Supply request updated!'})
 
-
-@app.route('/invitations/<int:invitation_id>', methods=['DELETE'])
-def delete_invitation(invitation_id):
-    invitation = Invitation.query.get(invitation_id)
-    if not invitation:
-        return jsonify({
-            "status": "Failed",
-            "message": "Invitation not found.",
-            "data": None
-        }), 404
-
-    db.session.delete(invitation)
+@app.route('/supply_requests/<int:request_id>', methods=['DELETE'])
+def delete_supply_request(request_id):
+    req = SupplyRequest.query.get_or_404(request_id)
+    db.session.delete(req)
     db.session.commit()
-
-    return jsonify({
-        "status": "Success",
-        "message": "Invitation deleted successfully.",
-        "data": None
-    }), 200
```

Comment:
These have also been removed. Kindly merge main onto your branch first before raising the PR so that you do not delete unnecessary stuff.